api/app/reviews/tests.py

```diff
@@ -1285,3 +1288,246 @@ def test_total_number_of_pages_when_zero(self):
 
         self.assertEqual(data['total_pages'], 0)
     
+
+class ReferenceReviewRequest(ApiTestCase):
+    def static_seed_data(self):
+        # User, country and organisation is set up by ApiTestCase
+        self.first_user = self.add_user('firstuser@mail.com', 'First', 'User', 'Mx')
+
+        event = self.add_event()
+
+        reviewer1 = AppUser('r1@r.com', 'reviewer', '1', 'Mr', password='abc', organisation_id=1, )
+        candidate1 = AppUser('c1@c.com', 'candidate', '1', 'Mr', password='abc', organisation_id=1, )
+        system_admin = AppUser('sa@sa.com', 'system_admin', '1', 'Ms', password='abc', organisation_id=1, is_admin=True)
+        event_admin = AppUser('ea@ea.com', 'event_admin', '1', 'Ms', password='abc', organisation_id=1)
+        users = [reviewer1, candidate1, system_admin, event_admin]
+        for user in users:
+            user.verify()
+        db.session.add_all(users)
+        db.session.commit()
+
+        event_roles = [
+            EventRole('admin', 4, 1),
+            EventRole('reviewer', 1, 1)
+        ]
+        db.session.add_all(event_roles)
+        db.session.commit()

```

Comment:
Please use `event.add_event_role()` instead (see [here](https://github.com/deep-learning-indaba/Baobab/blob/a5b4fa192dfe86869b7df01bcb5cbb8914b69b24/api/app/outcome/tests.py#L21) for an example)